CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## DFX
 
+### feat!: option for nodejs compatibility in dfx generate
+
+ca
```

Comment:
Looks like this is still todo

Reply:
I think I accidentally keyboard spammed while text was selected 🤦🏻